exercises/luhn/src/Luhn.hs

```diff
@@ -1,11 +1,16 @@
 module Luhn (addends, checkDigit, checksum, create, isValid) where
 
-addends = error "You need to implement this function."
+addends :: Int -> [Int]
+addends n = error "You need to implement this function."
```

Comment:
Yes. You should use `Integer` to avoid causing problems in some platforms. The Haskell standard's minimal range for `Int` is too short for our tests.